controller/controller.php

```diff
@@ -117,4 +124,32 @@ function contactMe() {
 }
 
 
+// Determine the posts list and page number to display them :
+function postsPager() {
+    
+    //obtain posts :
+    $postManager = new postManager();
+    $posts=$postManager->getList();
+
+    //Determine pages number to display 3 posts at a time
+    $posts=array_chunk($posts, 3);
+    $p_pages_nbr=array_key_last($posts)+1;
+    
+    // Return datas :
+    return $postsPager = [
+        'posts' => $posts, 
+        'p_pages_nbr' => $p_pages_nbr
+    ];
+}
+
+// Display blog posts :
+function blog($twig, $page, $p_pages_nbr, $posts, $action) {
+    
+    $offset=$page-1;
+    $posts=array_slice($posts, $offset, 1);
+
+    $posts=$posts[0];
+    echo $twig->render('blogView.html.twig', ['posts' => $posts, 'page' => $page, 'pages' => $p_pages_nbr, 'action' => $action]);
```

Comment:
For better readability, you should expand your array on multiple lines, same for the `render` globally.
```suggestion
    echo $twig->render(
        'blogView.html.twig',
        [
            'posts' => $posts,
            'page' => $page,
            'pages' => $p_pages_nbr,
            'action' => $action
        ]
    );
```

Reply:
Yes, you're right. I will modify this.